lua/lspconfig/efm.lua

```diff
@@ -8,7 +8,9 @@ local bin_name = "efm-langserver"
 configs[server_name] = {
   default_config = {
     cmd = {bin_name};
-    root_dir = util.root_pattern(".git");
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname) or vim.fn.getcwd()
```

Comment:
I think `util.path.dirname(fname)` might be better than `vim.fn.getcwd()` because it makes more sense if you do something like this:

```console
$ nvim ../../foo/bar/baz.py
```

```suggestion
      return util.root_pattern(".git")(fname) or util.path.dirname(fname)
```

Reply:
Damn, @mjlbach beat me to it 🤷‍♂️

Reply:
Convergent evolution :P

Reply:
Done. (Impressive feedback timing!)